.github/workflows/unit-tests.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           ([ -d "$COMPOSER_HOME" ] || mkdir "$COMPOSER_HOME") && cp .github/composer-config.json "$COMPOSER_HOME/config.json"
 
           echo COLUMNS=120 >> $GITHUB_ENV
-          echo PHPUNIT="$(pwd)/phpunit --exclude-group tty,benchmark,intl-data,integration" >> $GITHUB_ENV
+          echo PHPUNIT="$(pwd)/phpunit --exclude-group tty --exclude-group benchmark --exclude-group intl-data --exclude-group integration --fail-on-deprecation" >> $GITHUB_ENV
```

Comment:
I suggest configuring `--fail-on-deprecation` through the config file instead, so that it is also failing when running locally.

Reply:
done

Reply:
Looking at the failing high deps job I am afraid that we need to revert this again. Previously, we were able to silence deprecations setting the `SYMFONY_DEPRECATIONS_HELPER` env var to `weak`. There is no similar mechanism using PHPUnit (i.e. we cannot revert the `failOnDeprecation` attribute through a command-line flag if I don't miss anything.

Reply:
Those CLI settings would be perfect use case for negatable options to be able to override `true` values from the config files. This should be suggested to PHPUnit